src/source-af-packet.c

```diff
@@ -782,6 +782,7 @@ int AFPReadFromRing(AFPThreadVars *ptv)
             SCReturnInt(AFP_FAILURE);
         }
         PKT_SET_SRC(p, PKT_SRC_WIRE);
+        p->pkt_mode = PKT_MODE_AUTO;
```

Comment:
Since afpacket can work both in ips and ids mode, setting this means that it's allowed to do ips or ids on the packet if stream.inline is enabled or not.